packages/open-next/src/adapters/logger.ts

```diff
@@ -42,28 +42,33 @@ const isDownplayedErrorLog = (errorLog: AwsSdkClientCommandErrorLog) =>
 
 export function error(...args: any[]) {
   // we try to catch errors from the aws-sdk client and downplay some of them
-  if (
-    args.some((arg: AwsSdkClientCommandErrorLog) => isDownplayedErrorLog(arg))
-  ) {
+  if (args.some((arg) => isDownplayedErrorLog(arg))) {
     debug(...args);
-  } else if (args.some((arg) => arg.__openNextInternal)) {
+  } else if (args.some((arg) => isOpenNextError(arg))) {
     // In case of an internal error, we log it with the appropriate log level
-    const error = args.find(
-      (arg) => arg.__openNextInternal,
-    ) as BaseOpenNextError;
-    if (error.logLevel === 0) {
-      debug(...args);
+    const error = args.find((arg) => isOpenNextError(arg))!;
+    if (error.logLevel < getOpenNextErrorLogLevel()) {
       return;
     }
+    if (error.logLevel === 0) {
+      // Display the name and the message instead of full Open Next errors.
+      return debug(
+        ...args.map((arg) =>
```

Comment:
@conico974 the code does not display the full exception here and for level 1. Let me know what you think about that.

Reply:
That's good, as long as we have the message it's fine